pkgs/modules/go/default.nix

```diff
@@ -26,6 +26,7 @@ in
   replit.dev.formatters.go-fmt = {
     name = "go fmt";
     language = "go";
+    displayVersion = "Go ${go.version}";
```

Comment:
Why is this prefixed with "Go"?

Reply:
It's a "display version" so I used some leeway and put in what I wanted a user to see. But I think we are abandoning this approach in favor of going straight for Nixmodules V2.